lib/cfa/augeas_parser.rb

```diff
@@ -199,20 +248,22 @@ def load_value(aug, aug_key, keys_cache)
   end
 
   # @example read, print, modify and serialize again
-  #    require "config_files/augeas_parser"
+  #    require "cfa/augeas_parser"
   #
-  #    parser = CFA::AugeasParser.new("sysconfig.lns")
+  #    parser = CFA::AugeasParser.new("Sysconfig.lns")
```

Comment:
thanks for catching, maybe time to start using https://github.com/p0deje/yard-doctest ? or opposite way, filling examples with https://github.com/magec/yard-examples-from-rspec ?